src/EA.Weee.Web/Areas/Admin/Controllers/AddAatfController.cs

```diff
@@ -95,9 +95,10 @@ public async Task<ActionResult> SearchResults(string searchTerm)
         {
             SetBreadcrumb(InternalUserActivity.CreateAatf);
 
```

Comment:
Can the FetchSearchResultsJson method be removed? @willdot

Reply:
This is for the search as your type feature, so if that's enabled, it needs to stay in :)

Reply:
unfortunately this is existing code and didnt have unit tests... have just moved it from the scheme controller...

Reply:
The controller was taken from the external screens and looks like didn't originally have a test... will create some

Reply:
Added tests